instrumentation-genai/opentelemetry-instrumentation-google-genai/src/opentelemetry/instrumentation/google_genai/generate_content.py

```diff
@@ -317,7 +314,12 @@ def _maybe_update_error_type(self, response: GenerateContentResponse):
     def _maybe_log_system_instruction(
         self, config: Optional[GenerateContentConfigOrDict] = None
     ):
-        system_instruction = _get_config_property(config, "system_instruction")
+        system_instruction = None
```

Comment:
I thought `_get_config_property()` was doing this for you

Reply:
It was, but its main function was to support the span attributes that are now served with the dictionary flattening. The little bit of extra logic here is simpler than the `_get_config_property` definition, and so it seemed simpler overall to delete it in favor of the small bit of extra logic here.